source/developer/slash-commands.rst

```diff
@@ -3,111 +3,72 @@
-- **Mattermost slash commands are Slack-compatible.** If you have used Slack's slash commands previously to interact with external applications, you can reuse those same applications with Mattermost. Mattermost will automatically translate Slack's proprietary JSON payload format.
-- **Custom commands support auto-complete.** When you create a custom command for your teammates, you have the option to fill in information about how auto-complete should work with that command. This gives your teammates quick and easy access to use your custom slash command.
-
-**Example:**
-
-Suppose you have an external application that is able to check the weather for certain cities. By creating a custom slash command, and setting up the application to handle the HTTP POST or GET from the command, you could allow your users to check the weather in their city using your command. For example, a user might be able to type:
-
-``/weather toronto week``
-
-Your external weather application would receive an HTTP request letting it know the user is interested in Toronto and would like to know the weather for the week. The application could simply respond to the HTTP request with the following JSON payload:
-
-.. code-block:: Text
-
-  {"response_type": "in_channel", "text": "
-  ---
-  #### Weather in Toronto, Ontario for the Week of February 16th, 2016
-
-  | Day                 | Description                      | High   | Low    |
-  |:--------------------|:---------------------------------|:-------|:-------|
-  | Monday, Feb. 15     | Cloudy with a chance of flurries | 3 °C   | -12 °C |
-  | Tuesday, Feb. 16    | Sunny                            | 4 °C   | -8 °C  |
-  | Wednesday, Feb. 17  | Partly cloudly                   | 4 °C   | -14 °C |
-  | Thursday, Feb. 18   | Cloudy with a chance of rain     | 2 °C   | -13 °C |
-  | Friday, Feb. 19     | Overcast                         | 5 °C   | -7 °C  |
-  | Saturday, Feb. 20   | Sunny with cloudy patches        | 7 °C   | -4 °C  |
-  | Sunday, Feb. 21     | Partly cloudy                    | 6 °C   | -9 °C  |
-  ---
-  "}
-
-Which would render in a Mattermost message as follows:
-
-.. image:: ../images/weatherBot.PNG
-  :alt: Shows what the JSON payload renders as in Mattermost
+.. toctree::
+	:maxdepth: 2
 
 Built-in Commands
------------------
+------------------------
 
-Each Mattermost installation comes with some built-in slash commands that are ready to use. These commands are listed below:
+Each Mattermost installation comes with some built-in slash commands that are ready to use. These commands are available in the `latest Mattermost release <https://about.mattermost.com/download/>`_:
 
 .. csv-table::
     :header: "Command", "Description", "Example"
 
-    "/away", "Set your status to away", "/away"
+    "/away", "Set your status away", "/away"
```

Comment:
0/5 on this, but if we're grouping these, `/code`, `/echo`, `/me`, and `/shrug` could probably go together since they all print messages. We may also want to consider breaking the table up into sections so we can still keep them alphabetical, but it's not big enough that that would be a problem yet